report/styled_par.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * This example showcases the creation of styled paragraph
+ * The output is saved as styled_paragraph.pdf which illustrates some of the features
+ * of the creator.
+ */
+/*
+ * NOTE: This example depends on github.com/unidoc/unipdf/v3/creator, MIT licensed,
+ *       and github.com/unidoc/unipdf/v3/model,
+ *       Apache-2 licensed.
+ */
+
+package main
+
+import (
+	"log"
+
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func main() {
+
+	c := creator.New()
+	c.NewPage()
+
+	fontRegular, _ := model.NewStandard14Font(model.HelveticaName)
+	fontBold, _ := model.NewStandard14Font(model.HelveticaBoldName)
+
+	//Styled paragraphs ***********************************************
```

Comment:
Clean up comment style.  We don't use *** like this normally.  Keep consistent.  And there should be a space between the // and S...